var/spack/repos/builtin/packages/py-ml-collections/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class PyMlCollections(PythonPackage):
+    """ML Collections is a library of Python collections designed for ML usecases."""
+
+    homepage = "https://github.com/google/ml_collections"
+    pypi = "ml_collections/ml_collections-0.1.0.tar.gz"
+    git = "https://github.com/google/ml_collections"
+
+    version('0.1.0', commit='63a7c22d1bf3521c3bb3f9da6ebf07972c25fec4')
+
+    depends_on('python@2.6:', type=('build', 'run'))
+    depends_on('py-setuptools', type='build')
+    depends_on('py-absl-py', type=('build', 'run'))
+    depends_on('py-pyyaml', type=('build', 'run'))
+    depends_on('py-six', type=('build', 'run'))
+    depends_on('py-contextlib2', type=('build', 'run'))
+    depends_on('py-typing', type=('build', 'run'), when='^python@:3.6')
```

Comment:
```suggestion
    depends_on('py-typing', type=('build', 'run'), when='^python@:3.4')
```
Also missing dep on dataclasses